.github/workflows/rust.yml

```diff
@@ -9,6 +9,14 @@ jobs:
           - 1.34.0  # MSRV
           - stable
           - beta
+        features:
+          - ""
+          - "zeroize"
+        exclude: # Zeroize 1.1 supports Rust 1.39+
+          - rust: 1.34.0
+            features: "zeroize"
+    env:
+      features: ${{ }}
```

Comment:
I don't think it's required as it is now, right?

Reply:
My bad, it was just a test that I forgot.